odo/backends/sql.py

```diff
@@ -199,8 +199,10 @@ def discover_sqlalchemy_column(c):
 
 @discover.register(sa.sql.FromClause)
 def discover_sqlalchemy_selectable(t):
-    ordering = dict((c, i) for i, c in enumerate(c for c in t.columns.keys()))
+    ordering = dict((str(c), i)
```

Comment:
there is a function `__str__()`

I dunno... this seems longer... I'd prefer sticking with `str(c)`